apps/wallet-mobile/src/features/Receive/useCases/SpecificAmountScreen.tsx

```diff
@@ -0,0 +1,175 @@
+
+  const HEIGHT_SCREEN = useWindowDimensions().height
+  const HEIGHT_MODAL = (HEIGHT_SCREEN / 100) * 80
+
+  const [amount, setAmount] = useState<string>('')
+
+  const [isModalVisible, setIsModalVisible] = useState(false)
+
+  const [isCopying, copy] = useCopy()
+
+  const {colors, styles} = useStyles()
+
+  const currentAddress = _.last(receiveAddresses)
+
+  const generateLink = () => {
+    Keyboard.dismiss()
+    setIsModalVisible(true)
+  }
+
+  React.useEffect(() => {
+    wallet.generateNewReceiveAddressIfNeeded()
+  }, [wallet])
+
+  const {track} = useMetrics()
+
+  useFocusEffect(
+    React.useCallback(() => {
+      track.receivePageViewed()
+    }, [track]),
+  )
+
+  return (
+    <SafeAreaView style={{flex: 1}}>
+      <TouchableWithoutFeedback onPress={() => Keyboard.dismiss()}>
+        <KeyboardAvoidingView style={styles.root}>
+          <StatusBar type="dark" />
+
+          <Spacer height={24} />
+
+          <View style={styles.content}>
+            <View style={styles.screen}>
+              <Text style={styles.textAddressDetails}>{strings.specificAmountDescription}</Text>
+
+              <TextInput label={strings.ADALabel} keyboardType="number-pad" onChangeText={setAmount} maxLength={4} />
+
+              <View style={styles.textSection}>
+                <Text style={[styles.textAddressDetails, {color: colors.gray}]}>{strings.address}</Text>
+
+                <Text style={styles.textAddressDetails}>{currentAddress}</Text>
+              </View>
+            </View>
+
+            <Button
+              shelleyTheme
+              onPress={generateLink}
+              disabled={amount === '' ? true : false}
+              title={strings.generateLink}
+            />
+
+            <Spacer height={24} />
+          </View>
+        </KeyboardAvoidingView>
+      </TouchableWithoutFeedback>
+
+      {isModalVisible && (
+        <ModalScreenWrapper
+          title={strings.multipleAdress}
+          height={HEIGHT_MODAL}
+          onClose={() => {
+            setIsModalVisible(false)
+          }}
+        >
+          <View style={styles.root}>
+            <ScrollView>
+              {mocks.specificAddressAmount !== null ? (
+                <ShareQRCodeCard
+                  title={`${amount} ADA`}
+                  address={mocks.specificAddressAmount}
+                  onLongPress={() => copy(mocks.specificAddressAmount)}
+                />
+              ) : (
+                <View style={styles.root}>
+                  <SkeletonAdressDetail />
+                </View>
+              )}
+
+              <Spacer height={32} />
+
+              <Button
+                shelleyTheme
+                onPress={() => {
+                  copy(mocks.specificAddressAmount)
+                }}
+                disabled={amount === '' ? true : false}
+                title={strings.copyLinkBtn}
+                iconImage={require('../../../assets/img/copy.png')}
```

Comment:
no PNGs

Reply:
But button expect only .png and .jpeg images

Reply:
cc: @SorinC6